File: backend/routes/reviewRoutes.js
// cortate/backend/routes/reviewRoutes.js

const express = require('express');
const router = express.Router();
const auth = require('../middleware/authMiddleware');
const {
	crearResena,
	obtenerResenas,
	eliminarResena
} = require('../controllers/reviewController');

// Crear reseña (cliente)
router.post('/:barberoId', auth, crearResena);

// Obtener reseñas de un barbero
router.get('/:barberoId', obtenerResenas);

// Eliminar reseña (barbero)
router.delete('/eliminar/:id', auth, eliminarResena);

module.exports = router;

File: backend/routes/reviewRoutes.py
# cortate/backend/routes/reviewRoutes.py

from flask import Blueprint
from controllers.reviewController import review_controller

# Creamos el Blueprint de rutas para reseñas
review_bp = Blueprint('review_bp', __name__)

# Registramos las rutas del controlador dentro del blueprint de rutas
review_bp.register_blueprint(review_controller, url_prefix="/reviews")
